stograde/specs/cache.py

```diff
@@ -76,13 +76,40 @@ def convert_spec(yaml_path, json_path):
 def clarify_yaml(data):
     copied = copy.deepcopy(data)
     if 'files' in copied and copied['files'] is not None:
```

Comment:
What do you think about changing from `new_spec: true` to `spec_version: 2`?

Reply:
That sounds better, though how about `spec_version: 3` and make this a 3.0.0 update?

Reply:
Sure